check.py

```diff
@@ -23,13 +24,15 @@
 
 from scripts.test.support import run_command, split_wast, write_wast
 from scripts.test.shared import (
-    BIN_DIR, NATIVECC, NATIVEXX, NODEJS, WASM_AS,
-    WASM_CTOR_EVAL, WASM_OPT, WASM_SHELL, WASM_METADCE, WASM_DIS, WASM_REDUCE,
-    binary_format_check, delete_from_orbit, fail, fail_with_error,
-    fail_if_not_identical, fail_if_not_contained, has_vanilla_emcc,
-    has_vanilla_llvm, minify_check, options, tests, requested, warnings,
-    has_shell_timeout, fail_if_not_identical_to_file, with_pass_debug,
-    validate_binary
+    BIN_DIR, NATIVECC, NATIVEXX, NODEJS, WASM_AS, WASM_CTOR_EVAL, WASM_OPT,
+    WASM_SHELL, WASM_METADCE, WASM_DIS, WASM_REDUCE, CTOR_EVAL_TEST_DIR,
+    CRASH_TEST_DIR, EXAMPLE_TEST_DIR, METADCE_TEST_DIR, PASSES_TEST_DIR,
+    PRINT_TEST_DIR, REDUCE_TEST_DIR, SPEC_TEST_DIR, VALIDATOR_TEST_DIR,
+    WASM_BACKEND_TEST_DIR, binary_format_check, delete_from_orbit, fail,
+    fail_with_error, fail_if_not_identical, fail_if_not_contained,
+    has_vanilla_emcc, has_vanilla_llvm, minify_check, options, requested,
+    warnings, has_shell_timeout, fail_if_not_identical_to_file, with_pass_debug,
+    validate_binary, get_tests
```

Comment:
I'm not a big fan of these hub import statements.   I wonder if we should just `from scripts.test import shared` and then use `shared.` everywhere?

Reply:
After removing all `_TEST_DIR` I added, this import will remain mostly the same in this patch compared to the existing code, so I'm not sure it should be included in this patch. If we want that, I can make it a separate patch.

Reply:
No, no need to that as part of thls CL

Reply:
Done in #2462.